### ISSUE TITLE
json "message" field incorrect for "missing whitespace" error

### ISSUE THREAD

**Issue by infokiller** [hadolint/hadolint#593]
- [x] This is a bug report
- [x] I searched existing issues before opening this one

### Expected behavior

Clean error message in "message" field: `missing whitespace`

### Actual behavior

Error message mixed with other data: `/dev/stdin:1:4:\n  |\n1 | RUNNN\n  |    ^\nmissing whitespace\n`

### Steps to reproduce the behavior

```bash
docker run --rm -i hadolint/hadolint hadolint -f json - < <(echo 'RUNNN')
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v2.1.0-0-g85a5357
```

**Dockerfile** (if relevant)

```Dockerfile
RUNNN
```

**Additional environment details (OS, stack version, etc.)**

Linux 5.11.11-hardened1-1-hardened
